data/database/migration_system/data_conversion.py

```
@@ -0,0 +1,646 @@
+"""
+Data Conversion Logic for Social Download Manager
+
+Handles transformation of data between different schema versions,
+converting v1.2.1 flat structure to v2.0 normalized structure.
+"""
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from enum import Enum, auto
```

Ruff (0.11.9)

`enum.Enum` imported but unused

Remove unused import

(F401)